cli/internal/login/link.go

```diff
@@ -198,11 +198,7 @@ func (l *link) run() error {
 	}
 
 	if l.modifyGitIgnore {
-		fs.EnsureDir(".gitignore")
-		_, gitIgnoreErr := exec.Command("sh", "-c", "grep -qxF '.turbo' .gitignore || echo '.turbo' >> .gitignore").CombinedOutput()
-		if err != nil {
-			return fmt.Errorf("could find or update .gitignore.\n%w", gitIgnoreErr)
-		}
+		addTurboToGitignore()
```

Comment:
We want to make sure we handle the error here. In this case it looks like you can just return it